tests/test_repository.py

```diff
@@ -37,37 +55,49 @@ def test_gpg_signature_structure_is_preserved():
         "gpg_signature": ("filename.asc", "filecontent"),
     }
 
-    tuples = repository.Repository._convert_data_to_list_of_tuples(data)
+    tuples = repository.Repository._convert_metadata_to_list_of_tuples(data)
     assert tuples == [("gpg_signature", ("filename.asc", "filecontent"))]
 
 
-def test_content_structure_is_preserved():
-    """Preserve 'content' key when converting metadata."""
-    data = {
-        "content": ("filename", "filecontent"),
-    }
-
-    tuples = repository.Repository._convert_data_to_list_of_tuples(data)
-    assert tuples == [("content", ("filename", "filecontent"))]
-
-
 def test_iterables_are_flattened():
     """Flatten iterable metadata to list of tuples."""
     data = {
         "platform": ["UNKNOWN"],
     }
 
-    tuples = repository.Repository._convert_data_to_list_of_tuples(data)
+    tuples = repository.Repository._convert_metadata_to_list_of_tuples(data)
     assert tuples == [("platform", "UNKNOWN")]
 
     data = {
         "platform": ["UNKNOWN", "ANOTHERPLATFORM"],
     }
 
-    tuples = repository.Repository._convert_data_to_list_of_tuples(data)
+    tuples = repository.Repository._convert_metadata_to_list_of_tuples(data)
     assert tuples == [("platform", "UNKNOWN"), ("platform", "ANOTHERPLATFORM")]
 
 
+def test_form_metadata_translation(example_distribution):
```

Comment:
Again, you're not testing "translation" or serialization here, you're testing a private method which is already well tested above. I don't believe this test adds any real value.

Reply:
I removed this test.